sormas-app/app/src/main/res/layout/fragment_pathogen_test_read_layout.xml

```diff
@@ -73,6 +73,19 @@
 
                 </LinearLayout>
 
+                <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:orientation="horizontal">
+
+                    <de.symeda.sormas.app.component.controls.ControlTextReadField
```

Comment:
Typing ID here should also only be visible for the three pathogen test types; also, you don't need a wrapping LinearLayout when you use the ControlSingleColumnStyle